perf/lua/column_scan.lua

```diff
@@ -1,13 +1,15 @@
 --
```

Comment:
commit message to " perf: standardize an output of column_scan benchmark "

Reply:
In that case, the commit title is more than 50 symbols, so I shorten it to this one.